src/Microsoft.Azure.EventHubs/EventHubClient.cs

```diff
@@ -194,7 +194,12 @@ public static EventHubClient CreateFromConnectionString(string connectionString)
                 transportType);
         }
 
-        static EventHubClient Create(EventHubsConnectionStringBuilder csb)
+        /// <summary>
+        /// Creates a new instance of the Event Hubs client using the specified connection string builder.
```

Comment:
>Event Hubs client [](start = 42, length = 17)

<see cref="EventHubClient"/>

Reply:
Serkant says never mind the empty comment.